listenbrainz_spark/recommendations/create_dataframes.py

```diff
@@ -119,6 +127,56 @@ def generate_dataframe_id(metadata):
     metadata['dataframe_id'] = '{}-{}'.format(config.DATAFRAME_ID_PREFIX, uuid.uuid4())
 
 
+def unaccent_text(df):
```

Comment:
I have kept a generic name in case we want to unaccent some other text field in future.

Reply:
Hmmm. The function name is generic, but it actually unaccents two specific fields -- I wish that the function name suggested that it unaccents specific fields rather than a more general purpose unaccenting function.

Reply:
Oh, cool! Will do.